docs/middleware/ics29-fee/fee-distribution.md

```diff
@@ -0,0 +1,90 @@
+<!--
+order: 4
+-->
+
+# Fee distribution
+
+Learn about payee registration for the distribution of packet fees. The following document is intended for relayer operators. {synopsis}
```

Comment:
Nice. This is a meta-level comment but perhaps we can have who the document is intended for as its own seperate section at the top?